torch/testing/_internal/common_utils.py

```diff
@@ -1773,6 +1778,21 @@ def make_tensor(size, device: torch.device, dtype: torch.dtype, *, low=None, hig
         result = torch.repeat_interleave(result, 2, dim=-1)
         result = result[..., ::2]
 
+    if exclude_values:
+        if dtype in integral_types() or dtype is torch.bool:
+            replace_with = torch.tensor(1, device=device, dtype=dtype)
+        elif dtype in floating_types_and(torch.half, torch.bfloat16):
+            replace_with = torch.tensor(torch.finfo(dtype).eps, device=device, dtype=dtype)
+        else:
+            # An assert is required? It's already done above (L1766)
+            # assert dtype in complex_types()
```

Comment:
Is an assert required here? My thinking was that since it's already done before, so we don't need to do it again.

Reply:
Here we assume that the dtypes will be `complex`. However the branch is unconditional, so you can add
`assert dtype in complex_types()` as means to make sure our assumption stays true over time and errors out if it doesn't hold.

Reply:
Thanks, this makes sense. Fixed in recent commit.